### DISCUSSION TITLE
I get the error "GitHub Copilot could not connect to server. Extension activation failed. "Failed to get  copilot token"" every time I try install copilot on VS code

### DISCUSSION THREAD

**Discussion by TrippySkills** [community/community#14340] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/101980847/161814194-16cf08b9-91b9-4e34-bdcd-40c862691cab.png)

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by TrippySkills** [reply]
![image](https://user-images.githubusercontent.com/101980847/161822383-e7af4881-f32f-4b1a-9668-cb466abacc13.png)

**Reply by davecheney** [reply]
Please check your computers clock.

**Reply by TrippySkills** [reply]
I'm GMT
![image](https://user-images.githubusercontent.com/101980847/161822712-0ae0eaf6-91d8-41f3-ba8d-1d55dd71481a.png)

**Reply by davecheney** [reply]
Copilot issues a token that expires in 8 hours. Your computers clock is more than 8 hours ahead of the time our servers think it is. I think you may need to fix your clock to reflect your local time.

**Reply by TrippySkills** [reply]
😳 ok

**Reply by TrippySkills** [reply]
My clock seems okay, is there any other visible issue?

**Reply by davecheney** [reply]
From the screenshot you sent, your clock is more than 8 hours ahead of the correct time.

**Reply by TrippySkills** [reply]
Wow, from actual GMT time?

**Reply by TrippySkills** [reply]
I checked my phone, the time is the exact time on my pc as my phone.
